Automattic/wp-calypso
# Fix some missing elements in report list and re-wire the Cancel campa…

#### Proposed Changes

Updated Promoted Post report page to fix some issues with missing numbers. Now the Stop campaign button should point correctly to the DSP endpoint. Thumbnails are adding now the Photon CDN and once the widget is finished, the close button should work

#### Testing Instructions
- Go to https://wordpress.com/advertising/ and create a new campaign
- Once the campaign is finished, click on "Finish". It should return to the campaign list
- Go to "Campaign" list. Open a campaign summary. It should display values correctly
- Click on Cancel Campaign. The campaign should be stopped


#### Pre-merge Checklist
